src/Network/HTTP/Download/Verified.hs

```diff
@@ -59,9 +59,20 @@ data DownloadRequest = DownloadRequest
     , drRetryPolicy :: RetryPolicy
     }
 
--- | Default to retrying thrice with a short constant delay.
+-- | Default to retrying seven times with exponential backoff starting from
+-- one hundred milliseconds.
+--
+-- This means the tries will occur after these delays if necessary:
+--
+-- * 0.1s
+-- * 0.2s
+-- * 0.4s
+-- * 0.8s
+-- * 1.6s
+-- * 3.2s
+-- * 6.4s
 drRetryPolicyDefault :: RetryPolicy
-drRetryPolicyDefault = limitRetries 3 <> constantDelay onehundredMilliseconds
+drRetryPolicyDefault = limitRetries 7 <> constantDelay onehundredMilliseconds
```

Comment:
Should this be `constantDelay` or `exponentialBackoff`?

(If the latter it'll need importing, too, around line 33)

Reply:
Wauw, what a silly mistake. Expect a fix in about 2 mins.